docs/SRS/SRS.tex

```diff
@@ -217,6 +218,89 @@ \subsubsection{Other Stakeholders}
 
 \subsection{Mandated Constraints}
 
+\subsubsection{Solution Constraints}
+\emph{Description:} The product shall be built as a Progressive Web Application (PWA)\\
+\emph{Rationale:} The supervisor wants the application to be a PWA\\
+\emph{Fit Criterion:} The product shall be written using the Next.js PWA plugin
+
+\emph{Description:} The product shall be able to function on a variety of devices, such as on a computer, on tablets and on most modern phones\\
+\emph{Rationale:} Users will be accessing this product in a variety of scenarios, and will have access to different devices\\
+\emph{Fit Criterion:} The product shall be tested to function properly on Chrome's device toolbar, which includes the following devices:
+\begin{itemize}
+	\item iPhone SE
+	\item iPhone XR
+	\item iPhone 12 Pro
+	\item Pixel 5
+	\item Samsung Galaxy S8+
+	\item Samsung Galaxy S20 Ultra
+	\item iPad Air
+	\item iPad Mini
+	\item Surface Pro 7
+	\item Surface Duo
+	\item Galaxy Fold
+	\item Samsung Galaxy A51/71
+	\item Nest Hub
+	\item Nest Hub Max
+\end{itemize}
+However, due to timing constraints, testing will only be run on the most popular cases, which would include the iPhone, Pixel and Samsung phones, as well as iPad and Galaxy tablets.
+
+\subsubsection{Implementation Environment of the Current System}
+In the current design of the product, the product will be implemented in a cloud hosted serverless
```

Comment:
not sure if this counts but saying something like it should work with any web browser and any OS?

Reply:
Might be worth it for specifically the PWA aspect.